README.md

```diff
@@ -55,9 +55,17 @@ export interface ExampleSchema {
 
 ## Usage
 
+Check [sample app](sample/foo-typescript) for more details.
+
 ```js
 import {compileFromFile} from 'json-schema-to-typescript'
 fs.writeFileSync('foo.d.ts', await compileFromFile('foo.json'))
+
+async function generate() {
+  fs.writeFileSync('dist/foo.d.ts', await compileFromFile('src/foo.json'))
+}
```

Comment:
the duplicated block to remove it not the async function, but the one on line 62. 

It's not possible to call await outside a async function.